http-server-jetty/build.gradle

```diff
@@ -16,5 +21,5 @@ dependencies {
     testImplementation "io.micronaut.reactor:micronaut-reactor"
     testImplementation "io.micronaut.reactor:micronaut-reactor-http-client"
 
-    testImplementation "io.micronaut.security:micronaut-security:3.0.0-M2"
+    testImplementation("io.micronaut.security:micronaut-security:$micronautSecurityVersion")
```

Comment:
Does it work if we use the version in the bom?

Reply:
Yes...

Security 3.3.0 seems to rely on 3.3.0-M1 of core which seems wrong 🤔

https://ge.micronaut.io/s/tgm234sfxvk74/dependencies?focusedDependency=WzAsNyw0MjUsWzAsNyxbNDI1XV1d&toggled=W1swXSxbMCw3XSxbMCw3LFs0MjVdXV0

(the platform upgrades it to 3.3.0 though)